modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/init.es.js

```diff
@@ -16,9 +16,9 @@
 
 import {async} from 'metal';
 import {match} from 'metal-dom';
-import {utils, version} from 'senna';
-import globals from 'senna/lib/globals/globals';
 
+import globals from '../senna/globals/globals';
```

Comment:
Same deal here... instead of `globals/globals` would be nice to just have `globals`.

Reply:
I think we should probably remove `globals` alltogether... it smells rather badly. I think it might've been used to make this run in Node, maybe? Not sure the indirection makes sense at all